staging/src/k8s.io/client-go/tools/cache/shared_informer.go

```diff
@@ -476,6 +493,12 @@ func (s *sharedIndexInformer) HandleDeltas(obj interface{}) error {
 	return nil
 }
 
+// sharedProcessor has a collection of processorListener and can
+// distribute a notification object to its listeners.  Each distribute
+// operation is `sync` or not.  The sync distributions go to a subset
```

Comment:
> Each distribute operation is `sync` or not.

That sentence doesn't provide much value :)

Reply:
It's only job is to prepare the reader for two different behaviors of "distribute".  I have reworded, no longer focusing on a tautology.